pkg/cmd/root/help_topic.go

```diff
@@ -47,6 +47,26 @@ var HelpTopics = map[string]map[string]string{
 	},
 }
 
+func NewHelpTopicFn(topic, short string, fn func() string) *cobra.Command {
```

Comment:
What do you think about changing this name to `NewDynamicHelpTopic` and changing `NewHelpTopic` to `NewStaticHelpTopic`? My thoughts are that they are a bit more descriptive, and help make the distinction between the two a bit better.

Reply:
good idea